docs/reference/commandline/image.md

```diff
@@ -25,7 +25,7 @@ Options:
       --help   Print usage
 
 Commands:
-  build       Build an image from a Dockerfile
+  build       Build an image from a Dockerfile (same as 'docker build')
```

Comment:
This example shows the output how the CLI itself prints it, which does not include this).


We added additional metadata, and usage output in these PR's though (and will be using those in the documentation);

- https://github.com/docker/cli/pull/3693
- https://github.com/docker/cli/pull/3694